controllers/auctions.js

```diff
@@ -265,3 +265,26 @@ module.exports.selectWinner = async (req, res) => {
     throw error;
   }
 };
+
+module.exports.getBids = async (req, res) => {
+  const username = req.user.username;
+  const auctionId = req.params.id;
+
+  try {
+    const response = await axios.get(`${NE_BASE_URL}/rooms/${auctionId}`, {
+      auth: { username },
+    });
+
+    const allBids = response.data.Bids;
+    res.render('/auctions/showBids', { allBids });
+
+  } catch(error) {
+    if (error.isAxiosError) {
+      req.flash("error", error.response.data.message);
+    } else {
+      req.flash("error", "Failed to retrieve bid history.")
+    }
+    res.redirect(`/auctions/${auctionId}`);
```

Comment:
Om vi säger vi går in på `auction/<id>/bids` och vill få saker upplistade, om detta misslyckas beror det väl på ett serverfel och det borde visa typ 500 internal server error eller nåt. Inte redirecta och visa ett sånt meddelande? Eller bara jag som känner så?